pages/jobs.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { s3 } from 'common/constants/urls';
+import ZipRecruiterJobs from 'components/ZipRecruiterJobs/ZipRecruiterJobs';
```

Comment:
it's nitpicky, but could you import common components first, and then by appearance in the render after?

I'll document that logic in `CONTRIBUTING.md` or see if there's an eslint rule for it.

to be clear:
Section
FeaturedJobsData
FeaturedJobItem
ZipRecruiterJobs